src/main/java/com/everis/boundary/CarResources.java

```diff
@@ -75,15 +78,24 @@ public Response createCar(final Car car) {
     @Path("/{id}")
     public Response updateCar(final @PathParam("id") String id , final Car car) {
     	
-      LOGGER.info("Update new car!");
-    	try {
-			this.carService.updateCar(id,car);
-      LOGGER.info("Car Successfully Updated: " + car + "Id: " + id);
-			return Response.status(Status.OK).entity(car).build();
-		} catch (Exception e) {
-      LOGGER.error("Error: Car not found!");
-			return Response.status(Status.NOT_FOUND).entity("Car with id " + id + " not found").build();			
-		}	  
+    	car.setId(id);
+    	final ArrayList<String> validatorsErrors = ValidatorUtil.validate(car);
```

Comment:
Very nice! You're using the bean validation to check the car 👏 

But I would suggest a cleaner way using the annotation @Valid and afterwards returning an error message with an ExceptionMapper:   https://www.logicbig.com/tutorials/java-ee-tutorial/jax-rs/entity-validation.html